include/podio/UserDataCollection.h

```diff
@@ -206,6 +217,12 @@ class UserDataCollection : public CollectionBase {
   typename std::vector<BasicType>::const_iterator end() const {
     return _vec.end();
   }
+  typename std::vector<BasicType>::const_iterator cbegin() const {
```

Comment:
shouldn't we use here the definitions of line 80-84 instead of repeating?

Reply:
Thanks, `begin` , `cbegin`, `end`, `cend` use now type aliases